src/dev/build/tasks/os_packages/docker_generator/templates/dockerfile.template.js

```diff
@@ -32,7 +32,7 @@ function generator({ artifactTarball, versionTag, license, usePublicArtifact  })
   #
   # ** THIS IS AN AUTO-GENERATED FILE **
   #
```

Comment:
Apologies for the removal of the trailing whitespace -- I assumed this was to keep all indentation the same amongst lines. I did check if `dedent` handles this appropriately, which it does. For whatever reason my editor _really_ wants to remove these lines even when configured not to. But hey, think of all those bytes we save!

Reply:
👍  The [editorconfig](https://github.com/elastic/kibana/blob/master/.editorconfig#L9) calls for removing the whitespace